tests/wpunit/APITest.php

```diff
@@ -1228,278 +1228,469 @@ public function testTagUnsubscribeWithInvalidEmail()
 	}
 
 	/**
-	 * Test that the `get_subscriber_by_email()` function returns expected data
-	 * when valid parameters are provided.
+	 * Test that get_subscribers() returns the expected data.
 	 *
-	 * @since   1.0.0
+	 * @since   2.0.0
+	 *
+	 * @return void
 	 */
-	public function testGetSubscriberByEmail()
+	public function testGetSubscribers()
 	{
-		$this->markTestIncomplete();
+		$result = $this->api->get_subscribers();
 
-		$result = $this->api->get_subscriber_by_email($_ENV['CONVERTKIT_API_SUBSCRIBER_EMAIL']);
-		$this->assertNotInstanceOf(WP_Error::class, $result);
-		$this->assertIsArray($result);
-		$this->assertArrayHasKey('id', $result);
-		$this->assertEquals($result['id'], $_ENV['CONVERTKIT_API_SUBSCRIBER_ID']);
+		// Assert subscribers and pagination exist.
+		$this->assertDataExists($result, 'subscribers');
+		$this->assertPaginationExists($result);
 	}
 
 	/**
-	 * Test that the `get_subscriber_by_email()` function returns a WP_Error
-	 * when an empty $email parameter is provided.
+	 * Test that get_subscribers() returns the expected data
+	 * when the total count is included.
 	 *
-	 * @since   1.0.0
+	 * @since   2.0.0
+	 *
+	 * @return void
 	 */
-	public function testGetSubscriberByEmailWithEmptyEmail()
-	{
-		$this->markTestIncomplete();
+	public function testGetSubscribersWithTotalCount()
+	{
+		$result = $this->api->get_subscribers(
```

Comment:
These tests make me ask why we're not using named parameters?

Reply:
Because they're not supported in PHP 7.4 (only 8.0+).  As [~ 42% of WordPress sites are still using this version](https://en-gb.wordpress.org/about/stats/), sadly we need to run tests in PHP 7.4